Documentation/HowTo/adding-sections-and-pages.md

```diff
@@ -0,0 +1,61 @@
+# Adding Sections and Pages
+
+Publish works with two main types of pages: Sections and Pages.
+
+A **Section** is a top-level page which shows up in the menu.
+A **Page** is a stand-alone page which you can assign any URL. These do not show up in the menu.
+
+## Creating a new Section
+
+To create your sections, open up the main.swift file and fill in each section inside the `SectionID` enum like below:
+```
+struct MyWebsite: Website {
+  enum SectionID: String, WebsiteSectionID {
+    // Add the sections that you want your website to contain here:
+    case blog
+    case about
+  }
+}
+```
+
+### Adding content to your section-page
+
+To add content to these sections, you have a few options: markdown or manual Swift-code. To create your section using markdown, you must create a folder inside the Content-folder with the same name as your enum case. In the case above, you should create the folders: `/Content/blog` and `/Content/about`.
+
+To use a different folder name, you can assign a raw string value to the enum case like this: 
+```
+struct MyWebsite: Website {
+  enum SectionID: String, WebsiteSectionID {
+    case blog = "articles"
+    case about = "about-me"
+  }
+}
+```
+
+To create a page for the URL http://website.com/articles/ you must now create the folder `/Content/articles`.
+In this folder you should add an index.md file with the content you wish. This file will then serve as your sections "front-page".
+
+The file `/Content/articles/index.md` could look like so: 
+```
+# Articles
+
+Every week I write articles on the blog. See them in the list below.
+```
+
+With this added, run the project. Publish will now generate your site, and you can navigate to your new sections.
+
+## Creating a new Page
+
+Pages work exactly the same way as Sections, however, you should no longer add a case to the enum.
+
+Just like Sections you should add a top-level folder, and an index.md file inside, for example: `/Content/about/index.md` or `Content/contact/index.md`, with your desired contents.
+
+When you run Publish, URLs will be generated accordingly to generate the URLs: `http://website.com/about` and `http://website.com/contact` respectively.
```

Comment:
Is this part correct? If I make that folder structure the resulting URL is website.com/about/index.html. Is that expected or I'm doing something wrong here?

Reply:
What the docs say, and what you say is the same in my ears.
The url would be reachable from both ...about/ and ...about/index.html

Reply:
That's what I was expecting too. Is it possible that the local server doesn't behave that way?

See, with a test folder containing an index.md

<img width="214" alt="Screenshot 2020-03-10 at 19 30 27" src="https://user-images.githubusercontent.com/750912/76346624-9ab8a580-6305-11ea-8681-1176b1979398.png">

Going to `/test` doesn't work:

<img width="310" alt="Screenshot 2020-03-10 at 19 29 46" src="https://user-images.githubusercontent.com/750912/76346655-a6a46780-6305-11ea-852c-a137e0512e3e.png">

You need to go to`test/index/`

<img width="259" alt="Screenshot 2020-03-10 at 19 29 57" src="https://user-images.githubusercontent.com/750912/76346673-ac9a4880-6305-11ea-8026-ff4be2da9390.png">

Reply:
Hi @christiankm, even I'm seeing a similar behavior mentioned by @alexito4 . That is, say adding and `about` folder and an `index.md`, the page is viewed at `website.com/about/index.html`
May I know if you're able to achieve with just `website.com/about`?

Reply:
Never mind. it was as simple as adding `about.md` in the Content folder. 😅

Reply:
Hi guys, im having trouble working with resources and images. Does anyone know how to add images and logos to a theme and have it copy over to output?